resolwe/flow/models/entity.py

```diff
@@ -30,6 +30,20 @@ def duplicate(self, contributor, inherit_collection_membership=False):
             for entity in self
         ]
 
+    @transaction.atomic
+    def move_to_collection(self, source_collection, destination_collection):
+        """Move entities from source to destination collection."""
+        for entity in self:
+            # Remove from collection.
+            entity.collections.remove(source_collection)
+            for data in entity.data.all():
+                source_collection.data.remove(data)
```

Comment:
Can you just use `source_collection.data.remove(entity.data.all())`?

Reply:
`source_collection.data.remove(*entity.data.all())` will do yes.